tests/testdata/repo_03/repodata/repomd.xml

```diff
@@ -0,0 +1,66 @@
+    <timestamp>1533242352</timestamp>
+    <size>134</size>
+    <open-size>167</open-size>
+  </data>
+  <data type="filelists">
+    <checksum type="sha256">95a4415d859d7120efb6b3cf964c07bebbff9a5275ca673e6e74a97bcbfb2a5f</checksum>
+    <open-checksum type="sha256">bf9808b81cb2dbc54b4b8e35adc584ddcaa73bd81f7088d73bf7dbbada961310</open-checksum>
+    <location href="repodata/95a4415d859d7120efb6b3cf964c07bebbff9a5275ca673e6e74a97bcbfb2a5f-filelists.xml.gz"/>
+    <timestamp>1533242352</timestamp>
+    <size>123</size>
+    <open-size>125</open-size>
+  </data>
+  <data type="other">
+    <checksum type="sha256">ef3e20691954c3d1318ec3071a982da339f4ed76967ded668b795c9e070aaab6</checksum>
+    <open-checksum type="sha256">e0ed5e0054194df036cf09c1a911e15bf2a4e7f26f2a788b6f47d53e80717ccc</open-checksum>
+    <location href="repodata/ef3e20691954c3d1318ec3071a982da339f4ed76967ded668b795c9e070aaab6-other.xml.gz"/>
+    <timestamp>1533242352</timestamp>
+    <size>123</size>
+    <open-size>121</open-size>
+  </data>
+  <data type="primary_zck">
+    <checksum type="sha256">e0ac03cd77e95e724dbf90ded0dba664e233315a8940051dd8882c56b9878595</checksum>
+    <open-checksum type="sha256">e1e2ffd2fb1ee76f87b70750d00ca5677a252b397ab6c2389137a0c33e7b359f</open-checksum>
+    <header-checksum type="sha256">243baf7c02f5241d46f2e8c237ebc7ea7e257ca993d9cfe1304254c7ba7f6546</header-checksum>
+    <location href="repodata/e0ac03cd77e95e724dbf90ded0dba664e233315a8940051dd8882c56b9878595-primary.xml.zck"/>
+    <timestamp>1533242352</timestamp>
+    <size>269</size>
+    <open-size>167</open-size>
+    <header-size>132</header-size>
+  </data>
+  <data type="filelists_zck">
+    <checksum type="sha256">2e7db4492173b6c437fd1299dc335e63d09f24cbdadeac5175a61b787c2f7a44</checksum>
+    <open-checksum type="sha256">bf9808b81cb2dbc54b4b8e35adc584ddcaa73bd81f7088d73bf7dbbada961310</open-checksum>
+    <header-checksum type="sha256">f6232be7311fb31184a849895ae0030caf5b84e628b04be879dccf37e6ad2121</header-checksum>
+    <location href="repodata/2e7db4492173b6c437fd1299dc335e63d09f24cbdadeac5175a61b787c2f7a44-filelists.xml.zck"/>
+    <timestamp>1533242352</timestamp>
+    <size>262</size>
+    <open-size>125</open-size>
+    <header-size>131</header-size>
+  </data>
+  <data type="other_zck">
+    <checksum type="sha256">a939c4765106655c3f7a13fb41d0f239824efa66bcd6c1e6c044a854012bda75</checksum>
+    <open-checksum type="sha256">e0ed5e0054194df036cf09c1a911e15bf2a4e7f26f2a788b6f47d53e80717ccc</open-checksum>
+    <header-checksum type="sha256">482e0dcc2e58d0e616f9b00c7789948e6c41f89488e7f4ad2fc3bcc7b549c6d5</header-checksum>
+    <location href="repodata/a939c4765106655c3f7a13fb41d0f239824efa66bcd6c1e6c044a854012bda75-other.xml.zck"/>
+    <timestamp>1533242352</timestamp>
+    <size>263</size>
+    <open-size>121</open-size>
+    <header-size>131</header-size>
+  </data>
+  <data type="modules">
```

Comment:
I'm pretty sure modules can be `zck` compressed too, so we'll want tests for that.

Reply:
Do we need the zck content in the initial patch, or do I need to figure out how that works and get that done in this patch-set?